routes/store/order.py

```diff
@@ -11,6 +11,39 @@ def store_order_view(db_connection):
     orders = cursor.fetchall()
     return render_template('store/orders/view-orders.j2', orders_list=orders)
 
-@store_order_routes.route('/edit')
-def store_order_edit():
-    return render_template('store/orders/edit-order.j2')
+@store_order_routes.route('/edit/<int:id>', methods = ['GET', 'POST'])
```

Comment:
Can we get away with having the GET on `/edit` and the POST on a separate `/edit/<int:id>` route?